src-tauri/rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "nightly-2024-10-29"
```

Comment:
Pizza engine needs a nightly toolchain, which is required by the use of `hashbrown`